.github/workflows/release.yml

```diff
@@ -28,17 +34,39 @@ jobs:
       - name: Prepare environment
         env:
           PRIVATE_REPOSITORY_SBT: ${{ secrets.PRIVATE_REPOSITORY_SBT }}
-          SONATYPE_FILE: ${{ secrets.SONATYPE_FILE }}
-          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_SBT: ${{ secrets.SONATYPE_SBT }}
+          SONATYPE_PGP_SECRET: ${{ secrets.SONATYPE_PGP_SECRET }}
         run: |
```

Comment:
interogation: I always set env definition after run one, should be set a convention on our GA documentation about this?

Reply:
We definitively need a convention about that.

On my hand, there are two reasons for this order:
 * alphabetically `"env" < "run"`
 * you set environment variables before executing